synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.cfg

```diff
@@ -0,0 +1,30 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+  key: "STAGING_BUCKET_V2"
+  value: "docs-staging-v2"
+  # Production will be at: docs-staging-v2
+}
+
+# Configure the docker image for kokoro-trampoline
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java11"
+}
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
+  value: "github/java-memcache/.kokoro/release/publish_javadoc11.sh"
```

Comment:
If the 2 scripts are the same and the only differences are the docker image and the bucket name, we can re-use the same shell script.

Reply:
Actually it looks like the `destination-prefix` option is new. You could potentially make that configurable via an env variable or keep the 2 separate scripts.

Reply:
Makes sense, I think I'll just leave as is unless you have a strong opinion about this